frontend/interfaces/software.ts

```diff
@@ -43,7 +43,6 @@ export const TYPE_CONVERSION: Record<string, string> = {
   rpm_packages: "Package (RPM)",
   yum_sources: "Package (YUM)",
   npm_packages: "Package (NPM)",
-  atom_packages: "Package (Atom)",
```

Comment:
Similar thing to `frontend/pages/hosts/details/cards/Software/SoftwareTableConfig.tsx` here.